src/middlewared/middlewared/schema.py

```diff
@@ -195,7 +195,7 @@ def validate(self, value):
 
         verrors = ValidationErrors()
 
-        if value and len(value) > self.max_length:
+        if value and len(str(value)) > self.max_length:
```

Comment:
What would this be, if not str?

Reply:
`datetime.time` object in case of `Time` schema

Reply:
Almost seems like it should be handled in Time schema `validate` to keep things separate?

Reply:
I am on fence wrt doing this. I'll proceed with doing this in Time schema per your recommendation :)